### PR TITLE
Internal: Fix non blocking error on the Spell Check action

### PR DESCRIPTION
**Pull request by ronaldtebrake**
## Problem
```
Dictionary Error with (project-baseline)
        name: project-baseline
        msg: failed to load
        stack:
project-baseline: failed to load
    at /home/runner/work/_actions/streetsidesoftware/cspell-action/v2/action/node_modules/cspell-lib/dist/cjs/SpellingDictionary/DictionaryController/DictionaryLoader.js:110:147
    at async Promise.all (index 0)
```

Seems we didn't fully remove the baseline before.

## Solution
Make sure we remove the project baseline details from the cspell.json

## Issue tracker
None, internal. 

## How to test
See the error doesn't occur, as you can f.e. see here: https://github.com/goalgorilla/open_social/actions/runs/5422956794
Screenshot 2023-06-30 at 15 06 39

### Before merge
- [ ] Code/peer review is completed
- [ ] All commit messages are [clear and clean](https://open-social.slite.com/app/docs/DnmermZDIx_0OQ). If applicable a rebase was performed
- [ ] All automated tests are green
- [ ] Functional/manual tests of the acceptance criteria are approved
- [ ] All acceptance criteria were met
- [ ] New features or changes to existing features are covered by tests, either unit (preferably) or behat
- [ ] Update path is tested. New hook_updates should respect update order, right naming convention and consider hook_post_update code
- [ ] Module can be safely uninstalled. Update/implement hook_uninstall and make sure that removed configuration or dependencies are removed/uninstalled
- [ ] This pull request has all required labels (team/type/priority)
- [ ] This pull request has a milestone
- [ ] This pull request has an assignee (if applicable)
- [ ] Any front end changes are tested on all major browsers
- [ ] New UI elements, or changes on UI elements are approved by the design team
- [ ] New features, or feature changes are approved by the product owner

### After merge
- [ ] Code is tested on all branches that it has been cherry-picked
- [ ] Update hook number might need adjustment, make sure they have the correct order
- [ ] The Drupal.org ticket(s) are updated according to this pull request status

## Release notes
None, internal fix for the spell check action.

### CODE REVIEW

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3448-zosyaanexj76lzuss4pouq6eetrxhagq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/649ed37cdc23a4c553608178

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3448-zosyaanexj76lzuss4pouq6eetrxhagq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/649ed37cdc23a4c553608178

File: cspell.json [hunk 1]
```diff
@@ -30,6 +24,7 @@
   ],
   "ignorePaths": [
     "/cspell.words.txt",
+    "/cspell.json",
```
Comment (Kingdutch) [inline cspell.json:27]: Do we need to ignore the cspell.json? 🤔 That's not something I've seen in their docs

Comment (ronaldtebrake) [reply cspell.json:27]: Interestingly enough it seems so: (see previous commit run)
https://github.com/goalgorilla/open_social/actions/runs/5423242290/jobs/9860910946

**Review by Kingdutch — APPROVED** [review]

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3448-zosyaanexj76lzuss4pouq6eetrxhagq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/649ed37cdc23a4c553608178

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3448-zosyaanexj76lzuss4pouq6eetrxhagq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/649ed37cdc23a4c553608178

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3448-zosyaanexj76lzuss4pouq6eetrxhagq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/649ed37cdc23a4c553608178

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3448-zosyaanexj76lzuss4pouq6eetrxhagq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/649ed37cdc23a4c553608178

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3448-zosyaanexj76lzuss4pouq6eetrxhagq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/649ed37cdc23a4c553608178

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3448-zosyaanexj76lzuss4pouq6eetrxhagq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/649ed37cdc23a4c553608178

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3448-zosyaanexj76lzuss4pouq6eetrxhagq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/649ed37cdc23a4c553608178

**Pull request merged**